TodoList/src/app/screens/Checkbox/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { View, StyleSheet, TouchableOpacity } from 'react-native'
+import { styles } from './styles.js'
+
+export default class Checkbox extends Component {
+  render() {
+    const {onToggle, isChecked} = this.props
+    return (
+      <TouchableOpacity onPress={ onToggle }>
+        <View style={styles.box}>
+          { isChecked && <View style={styles.inner}/> }
```

Comment:
This is the wrong way to approach this. Use `isChecked` to determine the styles of the View. Ask me if you need help with this.